mobile-app/cypress/e2e/functional/wallet/auctions/auctions_bid_with_multiple_wallet.spec.ts

```diff
@@ -7,6 +7,8 @@ import {
   generateBlockUntilLiquidate,
 } from "../../../../support/walletCommands";
 
+BigNumber.set({ ROUNDING_MODE: BigNumber.ROUND_DOWN });
```

Comment:
I think there's a way for cypress to run this before all tests. Not sure

Reply:
Or does it makes sense to add it on all files?

Reply:
Was unable to find a way for cypress to run the config at the start. Tested the configurations if is not added to the files BigNumber would not round down